docs/configurations.md

```diff
@@ -76,4 +76,7 @@ summary: "Here we list all possible configurations and what they mean"
         - [usePrefix](#usePrefix) () <br/>
         <span style="color:grey">Standard prefix for all metrics</span>
 
+    - [S3Configs](s3_hoodie.html) (Hoodie S3 Configs) <br/>
```

Comment:
http://idratherbewriting.com/documentation-theme-jekyll/index.html fyi is what we use..  you can run it locally via commands in there